src/components/Sidebar/Sidebar.tsx

```diff
@@ -48,7 +72,19 @@ export function Sidebar() {
             </Button>
           </div>
           <ThemeSwitch />
-          <Input className="w-full" color="primary" placeholder="Search a country" variant="faded" />
+          <Select
+            placeholder="Search a country"
+            onChange={(event) => handleCountrySelect(event.target.value)}
+            className="w-full"
+            variant="faded"
+            color="primary"
+          >
+            {countryData.map((country) => (
+              <SelectItem key={country.name} value={country.name}>
+                {country.name}
+              </SelectItem>
+            ))}
+          </Select>
```

Comment:
I think it would be better if we used autocomplete here: https://nextui.org/docs/components/autocomplete